### PR TITLE
zeromq: build only 64bit, because dependent library: libsodium is alr…

### PR DESCRIPTION
**Pull request by klausz65**
…eady 64bit only.

### CODE REVIEW

File: components/library/zeromq/Makefile [hunk 1]
```diff
@@ -44,13 +44,10 @@ COMPONENT_PREP_ACTION = \
 
 COMPONENT_PRE_CONFIGURE_ACTION = ( $(CLONEY) $(SOURCE_DIR) $(@D) )
 
-# Build docs just once (for one ARCH), parameter varies from version to version
-CONFIGURE_OPTIONS.32 +=	--with-docs
-CONFIGURE_OPTIONS.64 +=	--without-docs
+CONFIGURE_OPTIONS +=	--with-docs
 CONFIGURE_OPTIONS +=	--enable-static=no
 CONFIGURE_OPTIONS +=	--with-libsodium
```
Comment (mtelka) [inline components/library/zeromq/Makefile:49]: I assume the other option would be to build 32-bit `zeremq` with `--without-libsodium` (or so).  But I do not know if we need 32-bit `zeremq` for anything.  If we don't then dropping 32-bit support here is okay.

Comment (klausz65) [reply components/library/zeromq/Makefile:49]: I checked all other libsodium packages and all of them are allready at 64bit:
network/megasync
network/chat/toxcore
service/network/dns/powerdns-recursor
library/c++/zeromq
web/php-81/php-common
mail/dovecot
library/python/pynacl-39
service/network/dns/dnscrypt-proxy
web/php-82/php-common
desktop/remote-desktop/remmina
editor/vim/vim-core
editor/gvim

and the only package which requires zeromq is: library/python/pyzmq-39 and this is of course also only 64bit.

Comment (mtelka) [reply components/library/zeromq/Makefile:49]: The other package that requires `zeromq` is `library/c++/czmq`.

Comment (klausz65) [reply components/library/zeromq/Makefile:49]: Okay, Marcel, I converted it to draft and will first do a 64 bit pull for library/c++/czmq - thanks for the heads up!

**Pull request merged**